misc/python/materialize/checks/roles.py

```diff
@@ -11,9 +11,13 @@
 
 from materialize.checks.actions import Testdrive
 from materialize.checks.checks import Check
+from materialize.util import MzVersion
 
 
 class CreateRole(Check):
+    def _can_run(self) -> bool:
+        return self.base_version >= MzVersion.parse_mz("0.45.0-dev")
```

Comment:
```suggestion
        return self.base_version >= MzVersion.parse("0.45.0-dev")
```
Sorry, I should have tested that.